### PR TITLE
Workaround to get parsepatch to run on alpine

### PR DESCRIPTION
**Pull request by La0**
Followup of #398 

Reported upstream on https://github.com/mozilla/parsepatch/issues/11

### CODE REVIEW

File: backend/code_review_backend/issues/management/commands/load_in_patch.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def load_hgmo_patch(diff):
     resp = requests.get(url)
     resp.raise_for_status()
 
-    patch = Patch.parse_patch(resp.content, skip_comments=False)
+    patch = Patch.parse_patch(resp.content.decode("utf-8"), skip_comments=False)
```
Comment (marco-c) [inline backend/code_review_backend/issues/management/commands/load_in_patch.py]: Might be better to ignore possible decoding errors

Comment (marco-c) [inline backend/code_review_backend/issues/management/commands/load_in_patch.py]:
```suggestion
    patch = Patch.parse_patch(resp.content.decode("utf-8", "ignore"), skip_comments=False)
```

Comment (jcristau) [reply backend/code_review_backend/issues/management/commands/load_in_patch.py]: why not `resp.text`?

Comment (marco-c) [reply backend/code_review_backend/issues/management/commands/load_in_patch.py]: Yep, that works too, it should be equivalent to resp.content.decode("utf-8", "replace").

Comment (La0) [reply backend/code_review_backend/issues/management/commands/load_in_patch.py]: i tested in alpine, that works nicely, thanks @jcristau !

**Pull request merged**